.vscode/launch.json

```diff
@@ -10,8 +10,9 @@
         "--extensionDevelopmentPath=${workspaceFolder}"
       ],
       "outFiles": [
-        "${workspaceFolder}/out/src"
-      ]
+        "${workspaceFolder}/out/**/*.js"
```

Comment:
I found this helped the debugger correctly pick up break points, guess it meant source maps where available but not sure